.cirrus.yml

```diff
@@ -41,7 +41,7 @@ debian-unstable_task:
     cpu: 1
   fix-pip_script:
     # see https://github.com/pypa/pip/pull/11623
-    - python3 -m pip install --upgrade "pip @ git+https://github.com/pypa/pip.git@a057c9bc99ad7eab2e3b1539e79eee7d9a923f4f"
+    - python3 -m pip install --upgrade pip>=23.0
```

Comment:
Nope. pip 23 is known to be broken on Debian. I tried my best, but failed: https://github.com/pypa/pip/pull/11740

Also, for some reason that I don't fully understand, it is not possible to upgrade the distribution provided pip on Debian unstable. I haven't had time yet to look into it.

Ah, and this would need some quotes, otherwise the shell will write the output of `python3 -m pip install --upgrade pip` into a file named `=23.0`.

Reply:
Thanks, I was trying to fix the Debian unstable CI too but failed :confused: